src/SavedData.js

```diff
@@ -0,0 +1,140 @@
+// Private Variables of SavedData
+let WEB_STORAGE_SUPPORT = (() => {
```

Comment:
We probably don't need _any_ of these `localStorage` checks anymore, since our lowest support level is IE 11.